app/bundles/ReportBundle/Tests/Form/DataTransformer/ReportFilterDataTransformerTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Mautic\ReportBundle\Tests\Form\DataTransformer;
+
+use Mautic\CoreBundle\Helper\DateTimeHelper;
+use Mautic\ReportBundle\Form\DataTransformer\ReportFilterDataTransformer;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
+use Symfony\Component\Form\Extension\Core\Type\DateType;
+use Symfony\Component\Form\Extension\Core\Type\TimeType;
+
+class ReportFilterDataTransformerTest extends TestCase
```

Comment:
All new classes should be final by default
```suggestion
final class ReportFilterDataTransformerTest extends TestCase
```